src/core/mount-setup.c

```diff
@@ -60,51 +60,51 @@ typedef struct MountPoint {
 #endif
 
 static const MountPoint mount_table[] = {
-        { "sysfs",       "/sys",                      "sysfs",      NULL,                      MS_NOSUID|MS_NOEXEC|MS_NODEV,
+        { "sysfs",       "/sys",                      "sysfs",      NULL,                                              MS_NOSUID|MS_NOEXEC|MS_NODEV,
           NULL,          MNT_FATAL|MNT_IN_CONTAINER },
-        { "proc",        "/proc",                     "proc",       NULL,                      MS_NOSUID|MS_NOEXEC|MS_NODEV,
+        { "proc",        "/proc",                     "proc",       NULL,                                              MS_NOSUID|MS_NOEXEC|MS_NODEV,
           NULL,          MNT_FATAL|MNT_IN_CONTAINER },
-        { "devtmpfs",    "/dev",                      "devtmpfs",   "mode=755",                MS_NOSUID|MS_NOEXEC|MS_STRICTATIME,
+        { "devtmpfs",    "/dev",                      "devtmpfs",   "mode=755,size=4M,nr_inodes=1k",                   MS_NOSUID|MS_NOEXEC|MS_STRICTATIME,
```

Comment:
hmm, M uppercase but k lowercase?

According to the tmpfs(5) man page lowercase is apparently correct

Reply:
So it's OK, like SI unit prefixes?

Reply:
Well, we have to work with what tmpfs documents...

Reply:
Ok, so M should be lowercase like tmpfs(5) says.